docs/docs/metrics.md

```diff
@@ -258,3 +258,126 @@ Dashes in metric names are replaced with underscores.
   state.
 * `marathon.threads.terminated.gauge` —
   the number of threads in `TERMINATED` state.
+
+## Alerting
+
+Let's consider a few examples of alerting rules. We will use Prometheus
+as our alerting tool, but please keep in mind that any alerting service
+of your choice can be used instead.
+
+Please note that particular threshold values are dependent on many
+variables like the number of applications, a cluster size and so on.
+
+For the syntax of Prometheus alerting rules please refer to
+[its official documentation](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
+
+Alert whenever a Marathon instance goes down or restarts:
+
+```yaml
+  - alert: Marathon Uptime
+    expr: up{job="marathon"} == 0
+    for: 30s
+    labels:
+      severity: warning
+    annotations:
+      summary: It fires when a Marathon instance stops or restarts.
+```
+
+Alert when there are less than 2 out of 3 Marathon instances that are
+up and running:
+
+```yaml
+  - alert: Marathon Too Few Instances
+    expr: sum(up{job="marathon"}) < 2
+    for: 30s
+    labels:
+      severity: critical
+      annotations:
+      summary: It fires when there are fewer Marathon instances up and running than expected.
+```
+
+Alert if Marathon's heap usage is too high:
+
+```yaml
+  - alert: Marathon Heap Usage
+    expr: avg_over_time(marathon_jvm_memory_heap_used_gauge_bytes[10s]) > (5 * 1024 * 1024 * 1024)
+    for: 10s
+    labels:
+      severity: critical
+    annotations:
+      summary: It fires when Marathon's heap is too big.
+```
+
+Alert if Marathon has too many threads:
+
+```yaml
+  - alert: Marathon Threads
+    expr: avg_over_time(marathon_jvm_threads_active_gauge[10s]) > 300
+    for: 10s
+    labels:
+      severity: critical
+    annotations:
+      summary: It fires when Marathon has too many threads.
+```
+
+Alert if Marathon dismisses deployments way too often:
+
+```yaml
+  - alert: Marathon Dismissed Deployments
+    expr: rate(marathon_deployments_dismissed_counter[1m]) > 3
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: It fires if there are too many deployments dismissed by Marathon.
+```
+
+Alert if Marathon receives offers, but is unable to use them for some
+reason, i.e. insufficient resources:
+
+```yaml
+  - alert: Marathon Given No Suitable Offer
+    expr: rate(marathon_mesos_offers_used_counter[1m]) == 0 and rate(marathon_mesos_offers_incoming_counter[1m]) > 0
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: It fires if there are incoming Mesos offers, but none can be used by Marathon.
+```
+
+Alert if there are too many concurrent HTTP requests to Marathon:
+
+```yaml
+  - alert: Marathon HTTP Requests
+    expr: avg_over_time(marathon_http_requests_active_gauge[10s]) > 100
```

Comment:
100 active requests per 10s are 10r/s. Three UI windows will probably produce that much ;)

Reply:
It is 100 requests per second on average during past 10 seconds.